### PR TITLE
Directed chatbubble icon to a user's commendations

### LINKED ISSUE #42: Top right box button should take you to MyCommendations
Note that in Next we don't use `<a href="">...</a>`, we use `<Link href="/me">...</Link>`

https://nextjs.org/docs/api-reference/next/link

### PR DESCRIPTION
**Pull request by ashlyndevries**
Closes #42

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -33,6 +35,6 @@ export const Header = () => {
 }
 
 // FIXME:
-// - align-content right rather than use ml (margin-left)
+// - align-content right rather than use ml (margin-left)??
```
Comment (SpBills) [inline components/Header.tsx:38]: nah

File: components/CommendationForm.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export const CommendationForm = ({ members }: { members: Array<Member> }) => {
 }
 
 // FIXME:
-// - tranform Select to AutoComplete?
+// - tranform Select to AutoComplete
 // - right-align CE member teams
 // - add avatar images {member.imageURL}
-// - disable button (funtionality: neither TextField nor Select can be null)
\ No newline at end of file
+// - disable button (funtionality: neither TextField nor Select can be null)?
```
Comment (SpBills) [inline components/CommendationForm.tsx:47]: yes

**Review by SpBills — COMMENTED** [review]
great job- waiting for Matthew to review

**Review by Konolo — APPROVED** [review]
LGTM

**Pull request merged**